pkg/occlient/occlient.go

```diff
@@ -1222,6 +1222,10 @@ func (c *Client) GetServerVersion() (*ServerInfo, error) {
 	return &info, nil
 }
 
+func (c *Client) IsOpenshift4() (bool, error) {
+	return c.GetKubeClient().IsResourceSupported("config.openshift.io", "v1", "clusterversions")
+}
+
```

Comment:
Is it alright to place this method here?
I can move this function to segment/context/context.go if it might not be used in other places.

Reply:
We should call it something related to the resource like `IsClusterVersionsSupported` similar to `IsProjectSupported`